src/HealthGPS/analysis_module.cpp

```diff
@@ -307,29 +307,28 @@ DALYsIndicator AnalysisModule::calculate_dalys(Population &population, unsigned
 // NOLINTBEGIN(readability-function-cognitive-complexity)
 void AnalysisModule::calculate_population_statistics(RuntimeContext &context,
                                                      DataSeries &series) const {
-    auto min_age = context.age_range().lower();
-    auto max_age = context.age_range().upper();
     if (series.size() > 0) {
         throw std::logic_error("This should be a new object!");
     }
 
     series.add_channels(channels_);
 
     auto current_time = static_cast<unsigned int>(context.time_now());
-    for (const auto &entity : context.population()) {
-        auto age = entity.age;
-        auto gender = entity.gender;
+    for (const auto &person : context.population()) {
+        auto age = person.age;
+        auto gender = person.gender;
 
-        if (!entity.is_active()) {
-            if (!entity.is_alive() && entity.time_of_death() == current_time) {
+        if (!person.is_active()) {
+            if (!person.is_alive() && person.time_of_death() == current_time) {
```

Comment:
Not relevant for this PR, but I feel like a `has_died()` method would be clearer than an `is_alive()` one.

Reply:
Then you'd have `is_alive` and `has_died =  !is_alive` functions?

Unless you mean like:

```c++
bool Person::has_just_died(unsigned int current_time) {
    return !is_alive() && time_of_death() == current_time;
}
```